config/application.yml.example

```diff
@@ -1,3 +1,44 @@
+# These settings are common across environments, but may be overridden
+# in any particular environment
+aamva_verification_request_timeout: '50'
+aamva_verification_url: https://verificationservices-cert.aamva.org:18449/dldv/2.1/online
+account_reset_wait_period_days: '0'
+acuant_timeout: '55'
+acuant_simulator: 'true'
+acuant_sdk_document_capture_enabled: 'true'
+allow_doc_auth_test_credentials: 'true'
+attribute_encryption_key_queue: '["old-key-one", "old-key-two"]'
+aws_region: us-west-2
+cac_proofing_enabled: 'true'
+dashboard_api_token: ''
+disallow_all_web_crawlers: 'true'
+disallow_ial2_recovery: 'false'
+doc_auth_enabled: 'true'
+doc_auth_exclusive: 'true'
+email_deletion_enabled: 'true'
+email_from: no-reply@identitysandbox.gov
+enable_usps_verification: 'true'
+enable_load_testing_mode: 'true'
+enable_mobile_capture: 'true'
+hmac_fingerprinter_key_queue: '["old-key-one", "old-key-two"]'
+in_person_proofing_enabled: 'false'
```

Comment:
this is not a dashboard feature flag as far as I can tell....should we remove it?